doc/release-notes-5807.md

```diff
@@ -0,0 +1,8 @@
+# Notable changes
+
+## HD Wallets Enabled by Default
+
+In this release, we are taking a significant step towards enhancing the Dash wallet's usability by enabling Hierarchical Deterministic (HD) wallets by default. This change aligns the behavior of `dashd` and `dash-qt` with the previously optional `-usehd=1` flag, making HD wallets the standard for all users.
+Dash Core still can be enforced by creating non-HD wallet by run with `-usehd=0` flag but this feature will be removed in next releases.
```

Comment:
feels like a duplicate
```suggestion
```

Reply:
@UdjinM6 
maybe, 

```
Command line argument `-usehd` is to remove in next major release
```
?

Reply:
I mean you already have

>However, users should be aware that this option is intended for legacy support and will be removed in future releases.

below, so why saying it again?